Doc/reference/datamodel.rst

```diff
@@ -1479,6 +1493,11 @@ Basic customization
    ``(x<y or x==y)`` does not imply ``x<=y``. To automatically generate ordering
    operations from a single root operation, see :func:`functools.total_ordering`.
 
+   By default, the :class:`object` class provides implementations consistent
+   with :ref:`comparisons`: equality compares according to object identity,
+   and order comparisons raise :exc:`TypeError`. Each default method may generate
+   these results directly, but may also return ``NotImplemented``.
```

Comment:
```suggestion
   these results directly, but may also return :data:`NotImplemented`.
```

IMO, the previous mentions (or at least the first instance) should be updated, but isn't strictly necessary in this PR.